src/orange/OrangeTrackView.hh

```diff
@@ -251,6 +279,51 @@ OrangeTrackView& OrangeTrackView::operator=(DetailedInitializer const& init)
     return *this;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * The current position
```

Comment:
End comments with `.` to enable the doxygen auto-brief docs. Also remove the `//!` from the declarations above in the class.